rules/windows/builtin/win_powershell_script_installed_as_service.yml

```diff
@@ -22,20 +29,29 @@ detection:
           - 'powershell'
           - 'pwsh'
     condition: selection and (selection1 or selection2)
+>>>>>>> 4f3a0efd88f27d846e0dda7a9b72f62c1296bc23
 falsepositives: Unknown
 level: high
 ---
 logsource:
     product: windows
     service: system
 detection:
+<<<<<<< HEAD
+    service_creation:
+=======
     selection:
+>>>>>>> 4f3a0efd88f27d846e0dda7a9b72f62c1296bc23
         EventID: 7045
 ---
 logsource:
     product: windows
     service: security
 detection:
+<<<<<<< HEAD
+    service_creation:
+        EventID: 4697
+=======
     selection:
```

Comment:
Hello @nsaddler!  Something bad happened during the commit. It looks like an unresolved conflict in git. Could you please recreate your PR with only your commit?

Reply:
Yes... Corrected it)